controller/test/parameters/parameters_test.cpp

```diff
@@ -0,0 +1,86 @@
+#include "parameters.h"
+#include "gtest/gtest.h"
+
+TEST(Parameters, SetAndGetRR) {
+  parameters_setRR(RR_MAX + 1);
+  float rr = parameters_getRR();
+  ASSERT_FLOAT_EQ(RR_MAX, rr);
```

Comment:
Nit, one-liner?

`ASSERT_FLOAT_EQ(RR_MAX, parameters_getRR())`

Same below.

Reply:
You're right! These should all be one liners. Thanks :D